### DISCUSSION TITLE
Menu Locks

### DISCUSSION THREAD

**Discussion by noor-codes** [linuxdeepin/developer-center#2838] [Features Request & Ideas | 特性请求 & 头脑风暴]
## Menu Lock

Need an option to lock the menu bar. sometimes I minimize the browser and accidentally resize it.
![image](https://user-images.githubusercontent.com/28804764/111287368-ca109980-8660-11eb-9a54-ce2c7e1586d0.png)

**Comment by noor-codes** [comment]
If there is also an option to completely hide my menu bar and only trigger the menu bar by clicking the `Window` button would be a massive help. 

Most of the time I don't wanna interact with it, I just want it to be hidden unless I press a key.

**Comment by justforlxz** [comment]
> If there is also an option to completely hide my menu bar and only trigger the menu bar by clicking the `Window` button would be a massive help.
> 
> Most of the time I don't wanna interact with it, I just want it to be hidden unless I press a key.

Does the always hidden mode not meet your needs?

**Comment by noor-codes** [comment]
Sure but I need a way to toggle it

**Comment by justforlxz** [comment]
> Sure but I need a way to toggle it

If you use the always hidden mode, move the mouse to the dock to temporarily display it, or press the super key to start the launcher. What kind of operation do you want?

**Comment by noor-codes** [comment]
- It’s critical to have lock functionality to stop the taskbar for accidentally resizing it while I just wanna resize a browser for example. 

- If there is a global shortcut to run which triggers the taskbar visibility would be a huge deal for me.

**Comment by noor-codes** [comment]
> > Sure but I need a way to toggle it
> 
> If you use the always hidden mode, move the mouse to the dock to temporarily display it, or press the super key to start the launcher. What kind of operation do you want?

Tried it, not helpful.

**Comment by meiyixiang** [comment]
Do you want shortcuts to call up the taskbar when it's hidden and hide it when it's not needed?

**Comment by noor-codes** [comment]
That would be awesome if there will be one
